### ISSUE TITLE
Training on Human3.6M does not converge

### ISSUE THREAD

**Issue by shamanneo** [neuroethology/BKinD-3D#6]
Hi, thank you for your excellent work.

I tried training the model using the preprocessed Human3.6M data along with config/H36M.yaml. However, even after many iterations, the loss seems to converge only around 1.7 and does not decrease further. Additionally, the saved visualizations indicate that the model is not learning properly — the predictions appear inaccurate or unstable.

Do you have any suggestions or known solutions for this issue?
Is there anything I should double-check in terms of training setup?

Thank you.

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/1390e6f2-901a-4953-a290-8252c9eff3d9" />

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/e43594ac-1ff4-4139-91f1-7072a3f589a9" />
